core/store/src/main/java/uk/gov/gchq/gaffer/store/SerialisationFactory.java

```diff
@@ -46,13 +46,13 @@
             new StringSerialiser(),
             new BytesSerialiser(),
             new CompactRawIntegerSerialiser(),
-            new RawIntegerSerialiser(),
             new CompactRawLongSerialiser(),
-            new RawLongSerialiser(),
             new BooleanSerialiser(),
```

Comment:
Please could you order these new serialisers so that the most commonly used serialisers occur at the top. A minor issue but it will be slightly faster. I would imagine that would be something like:

new OrderedRawLongSerialiser(),
new OrderedRawDateSerialiser(),
new OrderedRawIntegerSerialiser(),
new OrderedRawDoubleSerialiser(),
new OrderedRawFloatSerialiser()